### PR TITLE
Update fixture data for test site

### PR DESCRIPTION
**Pull request by sheralim012**
- Added Featured Content on Home Page
- Added Banner Index Page
- Added Banner Page in Banner Index Page
- Added Banner on Home Page

### CODE REVIEW

File: home/management/commands/create_initial_data.py [hunk 1]
```diff
@@ -37,6 +43,22 @@ def create(self, owner, home):
...
+        models.FeaturedContent.objects.create(source=home, content=youth)
+
+        banner_index_page = models.BannerIndexPage(title='Banners')
+        home.add_child(instance=banner_index_page)
+
+        image_url = 'https://via.placeholder.com/729x576.png?text=Youth'
+        http_res = requests.get(image_url)
+        title = 'youth_banner.jpg'
+        image_file = ImageFile(BytesIO(http_res.content), name=title)
+        image = Image.objects.create(title=title, file=image_file)
```
Comment (ehmadzubair) [inline home/management/commands/create_initial_data.py:55]: Better to move this to a function

Comment (sheralim012) [reply home/management/commands/create_initial_data.py]: addressed

**Comment by sheralim012** [comment]
Hey @istride, could you please go through this PR.

**Comment by ehmadzubair** [comment]
👍🏼

**Pull request merged**